### ISSUE TITLE
Bindings for a ViewModel List to the View

### ISSUE THREAD

**Issue by sialcasa** [sialcasa/mvvmFX#385]
There are several Usecases, where a Parent ViewModel manages ChildViewModels that should be displayed dynamically. Therefore I created an Binding for VM(s) to children Lists of Views. The first approach can be found here:

https://gist.github.com/sialcasa/42ce3f3310a56f8d2b6e91a8075ce166

**Comment by manuel-mauky** [comment]
This looks interesting and I can imagine a few use cases where this can be useful. 

it looks a bit similar to our utils for binding a List of ViewModels to a ListView (https://github.com/sialcasa/mvvmFX/wiki/ListView-containing-mvvmFX-views). 

We have the [CachedViewModelCellFactory](https://sialcasa.github.io/mvvmFX/javadoc/1.1.0/mvvmfx/de/saxsys/mvvmfx/utils/viewlist/CachedViewModelCellFactory.html) for this use case. 

When we include your util into the framework we should try to create a similar API like the existing CellFactory. Additionally we should think about caching. For ListViews this is needed because the CellFactory is called multiple times even if the underlying list hasn't changed. Is this needed for this use case too?

**Comment by manuel-mauky** [comment]
First I have to correct my previous comment. I've mentionend the caching that we have for ListViews. This isn't needed here because there is no CellFactory involved.

I've tried out your example and had a closer look at the code.
What this utility is actually doing can be seen as 2 steps:
1. `map` an `ObservableList` of ViewModels with a mapping function that loads a view for a given ViewModel. This results in a monadic ObservableList of loaded Views.
2. unidirectional bind the content of the mapped ObservableList to the children list of the layout container (for example `FlowPane`).

With [EasyBind](https://github.com/TomasMikula/EasyBind) you can achieve this behaviour like this:

``` java
import org.fxmisc.easybind.EasyBind;
import javafx.beans.binding.Bindings;  // normal JavaFX Bindings helper
...

ObservableList<Parent> views = EasyBind.map(viewModel.itemViewModels(), 
        vm -> FluentViewLoader.javaView(SubView.class).viewModel(vm).load().getView());

Bindings.bindContent(flowpane.getChildren(), views);
```

I've tried this out and as far as I can see it works exaclty like your utility.
So the question is whether we need our own implementation when there is a two-lines solution already existing? 
Maybe it's sufficient to describe the solution in our wiki.

**Comment by sialcasa** [comment]
Yes we should document your version - maybe we should introduce a documentation section about ways how to bind from the V to the VM where we can explain all the helpers like selectableItemList
